prime-dotnet-webapi/Models/PrivilegeType.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
+
+namespace Prime.Models
+{
+    [Table("PrivilegeTypeLookup")]
+    public class PrivilegeType : BaseAuditable, ILookup<short>
+    {
+        public readonly static short PHARMANET_TRANSACTIONS_TYPE = 2;
```

Comment:
A little nit-picky, but might as well drop the _TYPE from this. The use of it will read PrivilegeType.PHARMANET_TRANSACTIONS_TYPE which is redundant. 
Same for the group constants.